.builderrc

```diff
@@ -0,0 +1,3 @@
+---
+archetypes:
+  - builder-radium-component
```

Comment:
If we're using the Radium archetype, I'd expect this library to be part of the radium suite.

If it's going to be a tiny library, I'd rather it not use Builder and just bring its' own dependencies. Otherwise it should be a `radium-*` repository IMO.